lib/modules/modules.go

```diff
@@ -340,7 +351,7 @@ func ValidateResource(res types.Resource) error {
 	case types.AuthPreference:
 		switch r.GetSecondFactor() {
 		case constants.SecondFactorOff, constants.SecondFactorOptional:
-			return trace.BadParameter("cannot disable two-factor authentication on Cloud")
+			return trace.BadParameter("cannot disable two-factor authentication")
 		}
```

Comment:
This section can be removed now, right? Since the same check was already performed above?

Reply:
I think we want to leave it in while the environment variable is still there otherwise that check could get skipped